config-testnet.json

```diff
@@ -33,6 +33,11 @@
         "max-routes": 5,
         "min-osmo-liquidity": 0
     },
-    "enable-overwrite-routes-cache": true
+    "grpc-ingester":{
+        "enabled": false,
+        "max-receive-msg-size-bytes": 16777216,
```

Comment:
Note: 16 KB. We can't leave comments. Once we implement yml config, will be able to transcribe this better